Emby.Server.Implementations/Migrations/IMigrationRoutine.cs

```diff
@@ -1,6 +1,6 @@
 using System;
 
-namespace Jellyfin.Server.Migrations
+namespace Emby.Server.Implementations.Migrations
```

Comment:
These should all be in the `Jellyfin.Server.Implementations` namespace if being moved. We don't add to `Emby.Server.Implementations` unless we have a good reason to

Reply:
There are database migrations already in Jellyfin.Server.Implementations.Migrations, do you want to be able to differentiate between them and the ones being moved or just put it all in the same directory? Alternatively I could just not move them, I only did it as it was one of the suggestions.